### ISSUE TITLE
Build failure: gdm.passthru.tests.profile

### ISSUE THREAD

**Issue by OPNA2608** [NixOS/nixpkgs#341300]
### Steps To Reproduce

1. build `gdm.passthru.tests.profile`

### Build log

```
cat: /nix/store/i5bjyw7mgwl1938qnchskkhvg91cyxx3-gdm-46.2/share/dconf/profile/gdm: No such file or directory
GDM dconf profile changed, please update gdm.nix
```

### Additional context

Fails since at least #316402: https://github.com/NixOS/nixpkgs/runs/25684085838

### Notify maintainers

`teams.gnome.members`:
@bobby285271
@hedning
@jtojnar
@dasj19

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.45, NixOS, 24.05 (Uakari), 24.05.3884.4a92571f9207`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.90.0`
 - channels(bt1cn): `"unstable"`
 - channels(root): `"nixos-24.05, unstable"`
 - nixpkgs: `/home/puna/Development/nixpkgs`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by bobby285271** [comment]
The 47 PR has some early workarounds though I did not look deeper https://github.com/NixOS/nixpkgs/pull/333911/commits/a59897246bc32c9b6c6486676861b90667bae1bb

**Comment by jtojnar** [comment]
Yes, it got broken in GNOME 46 bump (https://gitlab.gnome.org/GNOME/gdm/-/commit/b0f802e36ff948a415bfd2bccaa268b6990515b7) – unfortunately I did not have time to give it thorough review.
